app/layout/aside_configure.phtml

```diff
@@ -1,3 +1,16 @@
+<?php
+function get_logout_url(): string {
+	if (($_SERVER['AUTH_TYPE'] ?? '') === 'openid-connect') {
+		$url_string = urlencode(Minz_Request::guessBaseUrl());
+		return PUBLIC_TO_INDEX_PATH . '/oidc/?logout=' . $url_string . '/';
```

Comment:
```suggestion
		return './oidc/?logout=' . $url_string . '/';
```

Changing my mind slightly: this should be a bit more robust for various types of installations including ones using sub-folders.